### PR TITLE
Z-Wave: Homeseer HS-WS100, HS-WD100, Aeotec Smart Dimmer 6 (ZW099)

### PR DESCRIPTION
**Pull request by savageautomate**
Z-Wave: 
- Added support for Homeseer HS-WS100 (Binary Switch)
- Update support for Homeseer HS-WD100 (Dimmer)
- Added support for Aeotec Smart Dimmer 6 (ZW099)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,3 +17,7 @@ distribution/openhabhome/etc/rrd4j/*.rrd
 *~
 features/*/src/main/history
 itemDump.txt
+
+# IntelliJ files
```
Comment (cdjackson) [inline .gitignore]: Please can you remove this change from this PR. If you want to update this file, then I'd recommend submitting it as a separate request.

**Comment by cdjackson** [comment]
I assume you've fully tested this? If so, please just update the .ignore file and I'll merge.

**Comment by savageautomate** [comment]
I'll submit a new PR with only the Homeseer devices.  I have tested the Homeseer devices pretty thoroughly or at least to the best of my ability as a newcomer to ZWave and OpenHAB.  Configurations, Associations and Commands seem to be working properly in OpenHAB v1.

I should probably test the Aeotec Smart Dimmer 6 (ZW099) a bit more.  The basic functionality is working.  However, I did not add this one to the database, but am wanting to incorporate it into my HA system so I'll work with it a bit more before submitting it.

Thanks, Robert

**Comment by savageautomate** [comment]
New PR: https://github.com/openhab/openhab/pull/4375

**Pull request closed without merging**